external-crates/move/crates/move-analyzer/editors/code/src/context.ts

```diff
@@ -19,7 +19,7 @@ function version(path: string, args?: readonly string[]): string | null {
     const versionString = childProcess.spawnSync(
         path, args, { encoding: 'utf8' },
     );
-    return versionString.stdout;
+    return versionString.stdout || null;
```

Comment:
This is unrelated change fixing a problem discovered when debugging `move-analyzer` changes for this PR